include/tvSettingsODM.h

```diff
@@ -56,92 +56,6 @@ extern "C"
 {
 #endif
 
-/**
- * @brief Gets supported video formats of the system. To be deprecated soon.
- * @note SOC vendors can stub these ODM functions and mark it as weak
- *
- * This function returns all the supported content formats. 
- *
- * @param[out] videoFormats    - Bitwise ORed values of enum ::tvVideoHDRFormat_t
- * @param[out] numberOfFormats - Count of supported video formats. Min should be 1
```

Comment:
Please check these structure / enum not used if not remove. Please check other also
tvhdr_type_t
tvWakeupSrcType_t,
tvVideoHDRFormat_t